pythonFiles/vscode_datascience_helpers/getVariableInfo/vscodeGetVariableInfo.py

```diff
@@ -9,9 +9,22 @@ def _VSCODE_getVariable(what_to_get, is_debugging, *args):
     collectionTypes = ["list", "tuple", "set"]
     arrayPageSize = 50
 
-    def truncateString(string):
+    def truncateString(variable):
+        string = _VSCODE_builtins.repr(variable)
         if _VSCODE_builtins.len(string) > maxStringLength:
-            return string[: maxStringLength - 1] + "..."
+            sizeInfo = (
+                "\n\nLength: " + str(_VSCODE_builtins.len(variable))
+                if _VSCODE_builtins.type(variable)
+                in [
+                    _VSCODE_builtins.str,
+                    _VSCODE_builtins.list,
+                    _VSCODE_builtins.tuple,
+                    _VSCODE_builtins.set,
+                    _VSCODE_builtins.dict,
+                ]
+                else ""
+            )
+            return string[: maxStringLength - 1] + "..." + sizeInfo
```

Comment:
the number of children would probably be more useful for collection types, which is already stored in the returned result. 
If I saw `['value1','value2', ... \n\n length: 100`, I would assume that length is the size of the array, so I assume copilot would too